src/components/pages/room.yml

```diff
@@ -1,12 +1,12 @@
-roomOwner:
-  en: Room owner
-  de: Raum Eigentümer
+privateRoomSubtitle:
+  en: Private room
+  de: Privatraum
```

Comment:
maybe also in German an adjective would sound better: "privater Raum"

Reply:
Changed